neurow/lib/neurow/public_api/endpoint.ex

```diff
@@ -53,7 +53,12 @@ defmodule Neurow.PublicApi do
 
         case last_event_id do
           :error ->
-            conn |> resp(:bad_request, "Wrong value for last-event-id")
+            conn
+            |> send_http_error(
```

Comment:
Error content needs to be sent as a SSE event, otherwise the frontend SSE client raises error becaose the response does not comply to the SSE protocol.